scripts/autogen

```diff
@@ -605,31 +605,34 @@ def gen_monolithic_undef_all_core(filt=None):
 
 def gen_monolithic_source_file(dry_run=False):
 
-    def fips202_only(c):
+    def fips202(c):
         return "fips202/" in c
 
-    def no_fips202(c):
-        return not fips202_only(c)
+    def randombytes(c):
+        return "randombytes.h" in c
+
+    def shared_headers(c):
```

Comment:
This is misnamed, we also consider `*.c` files

Reply:
fixed.